modules/deployment/src/test/java/org/mule/runtime/module/deployment/logging/LoggingSeparationTestCase.java

```diff
@@ -55,13 +63,13 @@ public void whenLoggingEachAppShouldLogOnItsOwnThreadSuccessfully() throws Excep
     executeApplicationFlow("logging");
 
     assertThat(logger.getAllLoggingEvents().size(), is(1));
-    // TODO: Change thread name check to TCCL check when available on slf4j-test
-    assertThat("", logger.getAllLoggingEvents().get(0).getThreadName(), stringContainsInOrder("logging-app-1"));
+    assertThat("", ((MuleArtifactClassLoader) logger.getAllLoggingEvents().get(0).getThreadContextClassLoader()).getArtifactId(),
+               stringContainsInOrder("logging-app-1"));
```

Comment:
I'm just noticing that we were using the `assertThat` version that takes a reason as the first parameter but we are passing it an empty string. Why is that?
If we are not interested in putting a custom reason string, let's use the version that takes only the actual and the matcher.